en/flight_controller/pixracer.md

```diff
@@ -2,35 +2,36 @@
 
 The Pixhawk<sup>&reg;</sup> XRacer board family is optimized for small racing quads and planes. In contrast to [Pixfalcon](../flight_controller/pixfalcon.md) and [Pixhawk](../flight_controller/pixhawk.md) it has in-built Wifi, new sensors, convenient full servo headers, CAN and supports 2M flash.
 
-> **Note** The main hardware documentation is here: https://pixhawk.org/modules/pixracer
-
 <img src="../../assets/flight_controller/pixracer/pixracer_hero_grey.jpg" width="300px" title="pixracer + 8266 grey" />
 
 
-## Quick Summary
+## Key Features
 
 * Main System-on-Chip: [STM32F427VIT6 rev.3](http://www.st.com/web/en/catalog/mmc/FM141/SC1169/SS1577/LN1789)
   * CPU: 180 MHz ARM Cortex<sup>&reg;</sup> M4 with single-precision FPU
   * RAM: 256 KB SRAM (L1)
 * Standard FPV form factor: 36x36 mm with standard 30.5 mm hole pattern
-* Wifi telemetry and software upgrade
+* Integrated Wifi telemetry and boot loading through the [ESP8266 WiFi Module](../telemetry/esp8266_wifi_module.md)
```

Comment:
The WiFi is not integrated, and not all pixracers come with it. Also `boot loading through the ESP8266 WiFi Module` is not possible.